R/fw_helpers_external.R

```diff
@@ -43,11 +43,25 @@ fw_create_header <- function() {
                                shiny::img(alt = "Working...",
                                           hspace = "5px",
                                           src = "img/loader.gif") ),
-            titleWidth = shiny::isolate(.g_opts$sidebar_size) )
+            titleWidth = shiny::isolate(.g_opts$sidebar_size))
     )
 }
 
-# Framework UI Sidebar Creation
+# Framework UI Header Creation that includes a right sidebar
+fw_create_header_plus <- function() {
+    return(
+        shinydashboardPlus::dashboardHeaderPlus(
+            title = shiny::div(class = "periscope-busy-ind",
+                               "Working",
+                               shiny::img(alt = "Working...",
+                                          hspace = "5px",
+                                          src = "img/loader.gif") ),
+            titleWidth = shiny::isolate(.g_opts$sidebar_size),
+            enable_rightsidebar = TRUE, rightSidebarIcon = "gears")
```

Comment:
can we make the icon a parameter the user can (optionally) specify?

Reply:
yes, good idea